docs/connector-development/connector-builder-ui/maintenance.md

```diff
@@ -0,0 +1,175 @@
+# Maintenance through Connector Builder
+
+This document will describe the process through which you can maintain a connector publicly available though our GitHub repository. This process has been split up in five broad steps:
+* Build
+* Merge into code
+* Validation (CATs)
+* Release
+* Operate/Maintain
+
+The visualization of this process is loosely maintained [here](https://whimsical.com/connector-maintenance-06-23-Rh2xAX5Byw641Xpf7P7qNg@2bsEvpTYSt1Hiz1WXyhZwMwZTwhXRsM2Sv8).
+
+## Build
+
+Goal: Modify the behavior of the connector
+
+Expected output: A manifest.yaml file that represents this new behavior
+
+All the steps mentioned below assume that you have the API available to perform requests. 
+
+### Validate Compatibility of Added Features
+Before implementing anything, we need to know if the features you want to add are supported by the Connector Builder. Please refer to the [compatibility guide](./connector-builder-compatibility.md) to confirm if you can build this change within the Connector Builder.
+
+### Config in Connector Builder
+
+Given that the connector already exists and that you want to use the account configured for CATs to iterate during the build phase, you can use [GSM](https://console.cloud.google.com/security/secret-manager?project=dataline-integration-testing) to fetch the existing configuration. Simply write the connector name in the filter bar, click on `SECRET_SOURCE-<name>__CREDS` and under `Actions`, select `View secret value` for the enabled version.
+
+### Importing in Connector Builder
+
+#### Connectors Without Manifest File
+
+This can be either a Python connector or a completely new one. Before implementing it in the Connector Builder, you should confirm if the API is suitable for the connector builder as per the [compatibility guide](./connector-builder-compatibility.md). Once this is confirmed, you are good to go! If not, please reach out to Airbyte to mention the incompatibility reason. This will help us prioritize changes and new features.
+
+#### Connectors With Manifest File
+Given a connector with a manifest.yaml file, you can attempt to import it. Go in the Connector Builder, click on "New custom connector" and select "Import a YAML". Given that the import is successful, you should be able to switch to the "UI" tab. Once this is done, there are a couple of manual validations to be performed:
+* Were the schemas properly imported? You can validate this by checking the `Declared schema` tab for a specific stream. If not, copy/paste the corresponding schema in the "Declared schema" tab. If the schemas have a "shared" folder, you will need to copy/paste the appropriate file whenever there is a `$ref` key.
```

Comment:
Because there can be a diff and I feel this is less error prone for the user to voluntarily update the schema seeing a diff than to just do a test run and without considering the current schema